src/tree/containerizedFunctionApp/ImageTreeItem.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { type Site } from "@azure/arm-appservice";
+import { AzExtParentTreeItem, GenericTreeItem, createContextValue, nonNullValue, type AzExtTreeItem, type TreeItemIconPath } from "@microsoft/vscode-azext-utils";
+import { ThemeIcon } from "vscode";
+import { localize } from "../../localize";
+
+
+export class ImageTreeItem extends AzExtParentTreeItem {
+    constructor(parent: AzExtParentTreeItem, public readonly site: Site, private readonly contextValuesToAdd: string[]) {
+        super(parent);
+    }
+
+    public async loadMoreChildrenImpl(): Promise<AzExtTreeItem[]> {
+        const imageInfo = this.site.siteConfig?.linuxFxVersion?.split('|')[1];
+        const loginServer = imageInfo?.split('/')[0];
+        const imageAndTag = imageInfo?.substring(nonNullValue(loginServer?.length) + 1, imageInfo?.length);
+
+        return [
+            new GenericTreeItem(this, {
+                label: localize('registry', 'Registry'),
+                description: loginServer,
+                contextValue: 'registry',
+                iconPath: new ThemeIcon('dash')
+            }),
+            new GenericTreeItem(this, {
+                label: localize('imageandTag', 'Image and Tag'),
```

Comment:
nit: I'm a little conflicted about this... I personally hate `Image and Tag` as the label (fully recognizing that this is what the portal does in the settings)

![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/468cd134-5ca7-45cd-9005-44980a9e05d8)

What do you think about just calling this node the "ID" or something? The image ID + tag is another identifier to point to a specific image (sort of like a commit) so I think that also makes sense.

Reply:
After in person discussion changing this to "Name" since Image is the title of the grouping.